### PR TITLE
Chat endpoints

### PR DESCRIPTION
**Pull request by mukobi**
Get chat endpoint was already written in #16, and I pretty much copied the logic from #12 with a few modifications. Name defaults to "anonymous" if not specified (including empty string).

### CODE REVIEW

File: src/controllers/chatController.ts [hunk 1]
```diff
@@ -21,6 +21,20 @@ export const chatController = {
...
     res.send(await getChatFromDB(gameIndex));
   },
+  /**
+   * Get all chat messages for the game_name specified.
+   * Sends the results as JSON.
+   * @param req Request object
+   * @param res Response object
+   */
+  async postToChat(req: express.Request, res: express.Response) {
+    const internalGameName: string = req.params['game_name'];
```
Comment (evankanderson) [inline src/controllers/chatController.ts]: Do you need "internal" in this name? It doesn't seem like there's any other "game name" in this function.

File: src/controllers/chatController.ts [hunk 2]
```diff
@@ -45,3 +59,38 @@ async function getChatFromDB(gameIndex: number): Promise<object[]> {
     throw err;
   }
 }
+
+/**
+ * Returns the current UTC time as a readable string.
+ * E.g. "2019-08-19 23:53:49"
+ */
+function getCurrentTime(): string {
```
Comment (evankanderson) [inline src/controllers/chatController.ts]: I think we have another copy of this function in the high score code. Should we unify them?

File: src/controllers/chatController.ts (hunk 1 above)
Comment (mukobi) [reply src/controllers/chatController.ts]: Done https://github.com/knative-portability/Kubercade/pull/20/commits/7f706ae8393b9d7d08195a8f38616a4edece0d4e. It was originally copied from `scoresController.ts` which uses both `internalGameName` (pac-man) and `prettyGameName` (Pac-Man).

File: src/controllers/chatController.ts (hunk 2 above)
Comment (mukobi) [reply src/controllers/chatController.ts]: I opted to remove this with the schema change in #22.

**Comment by mukobi** [comment]
Ready for more review.

File: src/controllers/chatController.ts (hunk 2 above)
Comment (evankanderson) [reply src/controllers/chatController.ts]: (waiting to review this part until #22 is merged)

**Comment by mukobi** [comment]
Closing since #22 implements these changes and more better.

**Pull request closed without merging**